src/rpc/methods/eth.rs

```diff
@@ -211,6 +211,10 @@ impl EthUint64 {
                 .try_into()?,
         )))
     }
+
+    pub fn to_hex_string(&self) -> String {
+        format!("0x{:x}", self.0)
+    }
```

Comment:
nit: as far as I recall, the `format!` performance is not great, especially compared to dedicated crates like `hex`. I'd rather we use the latter.

It might make sense to consider https://lib.rs/crates/faster-hex in the future to replace the `hex` as well.